### PR TITLE
Update README.md with more details

### PR DESCRIPTION
**Pull request by aghassemi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,26 @@
 # wg-ui
-Responsible for AMP's visual components &amp; interactions and AMP's overall user experience (including published guidelines for AMP UX). The UI WG also has overall responsibility for ensuring AMP is accessible, though each WG is responsible for ensuring accessibility within their areas of responsibility. Facilitator: @aghassemi
+**wg-ui** [Working Group](https://github.com/ampproject/meta/blob/master/GOVERNANCE.md#working-groups) is responsible for AMP's visual components &amp; interactions and AMP's overall user experience (including published guidelines for AMP UX). The UI Working Group also has overall responsibility for ensuring AMP is accessible, though each WG is responsible for ensuring accessibility within their areas of responsibility. Facilitator: @aghassemi
```
Comment (nainar) [inline README.md]: New line before Facilitator

File: README.md [hunk 2]
```diff
@@ -1,2 +1,26 @@
...
+
+# Members
+- @aghassemi - Ali Ghassemi (Facilitator)
+- @alanorozco - Alan Orozco
+- @andrewwatterson - Andrew Watterson
+- @cathyxz - Cathy Zhu
+- @cvializ - Carlos Vializ
+- @nainar - Naina Raisinghani
+- @sparhami - Sepand Parhami
```
Comment (nainar) [inline README.md]: Should include Jonathan?

Comment (aghassemi) [reply README.md]: good point. I am following up with him. will add later

File: README.md (hunk 1 above)
Comment (aghassemi) [reply README.md]: done

**Comment by mrjoro** [comment]
LGTM; thanks!

**Pull request merged**